src/services/codefixes/convertToEs6Module.ts

```diff
@@ -524,4 +572,16 @@ namespace ts.codefix {
             exportSpecifiers && factory.createNamedExports(exportSpecifiers),
             moduleSpecifier === undefined ? undefined : factory.createStringLiteral(moduleSpecifier));
     }
+
+    interface ConvertedImports {
+        newImports: readonly Node[];
+        useSitesToUnqualify?: ESMap<Node, Node>;
+    }
+
+    function convertedImports(newImports: readonly Node[], useSitesToUnqualify?: ESMap<Node, Node>): ConvertedImports {

```

Comment:
This function seems a bit superfluous to me; there's only one call with a second argument.
I guess it lets you avoid writing casts or explicit return types in some places?

Reply:
It was because of all the places that _didn’t_ have the second argument that I added this; all those were one-liner returns that would have become four lines in order to add the object literal with `useSitesToUnqualify: undefined` on a line.